### PR TITLE
Ship animation

### PR DESCRIPTION
**Pull request by vlad2876**

### CODE REVIEW

File: src/app/components/home/game-container/skyline-area/skyline-area.component.ts [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+  ]
 })
 export class SkylineAreaComponent implements OnInit {
 
   ShipType = ShipType;
 
+  shipMovementState = 'start';
+
+  shipMovementAnimation = setInterval(() => {
+    this.shipMovementState = 'end';
```
Comment (Zmitserkos) [inline src/app/components/home/game-container/skyline-area/skyline-area.component.ts:24]: Move these values to an enum.

File: src/app/components/home/game-container/skyline-area/skyline-area.component.ts [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+    setTimeout(() => {
+      this.shipMovementState = 'start';
+    }, 5100);
```
Comment (Zmitserkos) [inline src/app/components/home/game-container/skyline-area/skyline-area.component.ts:27]: Magic number

File: src/app/components/home/game-container/skyline-area/skyline-area.component.ts (hunk 1 above)
Comment (vlad2876) [reply src/app/components/home/game-container/skyline-area/skyline-area.component.ts]: Fixed

File: src/app/components/home/game-container/skyline-area/skyline-area.component.ts (hunk 2 above)
Comment (vlad2876) [reply src/app/components/home/game-container/skyline-area/skyline-area.component.ts]: Fixed

File: src/app/components/home/game-container/game-container-enums/ship-state.enum.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export enum ShipStateEnum {
+  Start = 'start', End = 'end'
```
Comment (Zmitserkos) [inline src/app/components/home/game-container/game-container-enums/ship-state.enum.ts:2]: Please put End = 'end' on a new line.

File: src/app/components/home/game-container/game-container-enums/ship-state.enum.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export enum ShipStateEnum {
```
Comment (Zmitserkos) [inline src/app/components/home/game-container/game-container-enums/ship-state.enum.ts:1]: Minor. It's better not to use Enum in the name, but it's a matter of preference.

**Review by Zmitserkos — APPROVED** [review]

File: src/app/components/home/game-container/game-container-enums/ship-state.enum.ts (hunk 3 above)
Comment (vlad2876) [reply src/app/components/home/game-container/game-container-enums/ship-state.enum.ts]: Fixed

File: src/app/components/home/game-container/game-container-enums/ship-state.enum.ts (hunk 4 above)
Comment (vlad2876) [reply src/app/components/home/game-container/game-container-enums/ship-state.enum.ts]: Fixed

**Pull request merged**